src/main/java/frc/robot/subsystems/DiffDriver.java

```diff
@@ -0,0 +1,92 @@
+package frc.robot.subsystems;
```

Comment:
Just for consistency across files, add a blank line at the beginning and at the end of every file.